### ISSUE TITLE
Is redirecting url supported in modheader_selenium? 

### ISSUE THREAD

**Issue by kxusl** [modheader/modheader_selenium#31]
Hi,

Redirect URL is supported in the modHeader. I wonder if it is supported in the modeheader_selenium as well. I could not find it in the source code.

Thanks,
Kevin

**Comment by anthonybaldwin** [comment]
I'm not sure on the answer, so sorry if this is unhelpful. But I'm generally curious what the use case/test is with Selenium? 🤔

My initial reaction was: "What's the point of setting **A** -> **B** via modheader_selenium, then having your program to navigate to **A**? Why not just navigate straight to **B**?"

**Comment by hao1300** [comment]
Redirect URL is not currently supported in modheader_selenium. Regrettably many features that go into ModHeader extension has not been ported to modheader_selenium, mainly because there was not a strong demand for it.

Are you trying to get <a.com/url> to redirect to <b.com/url> as a HTTP 302 redirect, or are you trying to proxy <a.com/url> request to <localhost/url> while keeping the browser URL unchanged?

**Comment by kxusl** [comment]
Thanks for the reply. I want to replace <a.com/v1/url> to <a.com/dev/url> so the frontend code does not need to change when testing with the dev stage API. I appreciate it if you can give me some good suggestions. I have used modHeader in my browser to do it but I also want to use Selenium to automate the test case. Can I use modHeader directly for selenium?

**Comment by hao1300** [comment]
You can use ModHeader directly for Selenium, but it is not optimized for automated usage, and may not work the way you would expect it to work. I can take a look at adding support for this into modheader_selenium next week.

**Comment by hao1300** [comment]
@kxusl - good news. I was able to put the webdriver codes into ModHeader extension, gated behind some special flag that is only enabled for the webdriver build. I have updated the CRX and XPI files with the latest change. They are much bigger than before, but now pretty much all features available in ModHeader should be available in WebDriver as well. You will just need to export your profile and load it into webdriver using the `/load` API.
